index.html

```diff
@@ -144,9 +144,78 @@
     <h1>Introduction</h1>
 
     <p>
-This document serves as an official registry for all known global parameters,
+This document serves as an official registry for all known methods, parameters,
```

Comment:
Consider "all well-known" instead of "all known", since there may be many extensions that are not registered.

Reply:
accepted